Debugtest.md

```diff
@@ -0,0 +1,49 @@
+# How to Debug Test Failures:
+
+1. When the test fails, open the target folder where the reports are generated after execution.
+2. Expand the cucumber-reports>advanced-reports>cumumber-html-reports>js>overview-failures.html file.
+3. Open the report in browser and check the fail step.
+4. Now go to the feature file where the step failed.
+5. Press command key and click on the step failed.
+6. This redirects to the steps design.
+7. We may always select the step to be executed in Debug mode, to figure out what went wrong.
+8. Define where the program needs to be stopped. This is done using breakpoints. Breakpoints are special markers, which represent places and/or conditions when the debugger needs to step in and freeze the program state. A program, which has been frozen by the debugger is referred to as suspended.
+
+## Step Actions:
+
+There are different ways to navigate the code in debug mode. For example, we might prefer to execute a line of code without bothering about the details of the methods being called. We can do this through the various step actions.
+
+Set a breakpoint before we start the application in the debug mode. 
+
+**The various step actions are:**
```

Comment:
This section is not needed. Users will want to know how to debug test failures, not how to use Intellij debugger.

Reply:
Removed.